src/main/com/yetanalytics/squuid.cljc

```diff
@@ -1,5 +1,6 @@
 (ns com.yetanalytics.squuid
   (:require [clojure.spec.alpha :as s]
+            [clojure.spec.gen.alpha :as sgen]
```

Comment:
`sgen` is only used in clj mode, not cljs. Normally this wouldn't be a big deal beyond a "unused require" warning, but in this case it's actually important since `sgen` doesn't work in cljs without additional requires.

Reply:
Sorry about that!